app/assets/javascripts/application.js

```diff
@@ -24,11 +25,40 @@ $(document).ready(function() {
 function sellItem(id){
     $(id).toggleClass('hide');
     $(id).toggleClass('show');
+    findSellTotal();
+    updateTotal();
+}
+
+function findSellTotal(){
+  var sold_items = $("ul.character_trades").find(".show")
+  items_total = [0]
+  $.each(sold_items, function(index, sold_item){
+      items_total.push(parseInt($(this).find('.price').text()))
+  })
+  total = _.sum(items_total);
+  $( ".sold_value" ).text( total );
 }
 
 function buyItem(id){
     $(id).toggleClass('hide');
     $(id).toggleClass('show');
+    findBoughtTotal();
+    updateTotal();
+}
+
+function findBoughtTotal(){
+  var bought_items = $("ul.store_trades").find(".show")
+  items_total = [0]
+  $.each(bought_items, function(index, bought_item){
```

Comment:
Identifier 'bought_items' is not in camel case.<br>Identifier 'bought_item' is not in camel case.